server/shared/auth/authenticator.js

```diff
@@ -17,10 +21,46 @@ class Auth extends Authenticator {
     return super.initialize(options);
   }
 
-  authenticate(strategy, { failWithError = true, ...options } = {}) {
+  authenticate(strategy, options, callback) {
+    if (isFunction(options)) {
+      callback = options;
```

Comment:
This is c/p from [passport source code](https://github.com/jaredhanson/passport/blob/master/lib/middleware/authenticate.js#L71) but I agree that it can be polished 👍